src/me/mrCookieSlime/Slimefun/listeners/BackpackListener.java

```diff
@@ -65,7 +66,7 @@ public void onClick(InventoryClickEvent e) {
 			else {
 				SlimefunItem sfItem = SlimefunItem.getByItem(e.getCurrentItem());
 				if (SlimefunManager.isItemSimiliar(item, SlimefunItem.getItem("COOLER"), false)) {
-					if (e.getCurrentItem() == null);
+					if (e.getCurrentItem().getType() == Material.AIR);
```

Comment:
I would probably do (just to be sure)

== null || getType().equals(Material.AIR)

Also, it is good practice to use .equals instead of == when comparing non-primitives, especially enums.